test/mocks/StakingRewardsMock.sol

```diff
@@ -0,0 +1,203 @@
+// SPDX-FileCopyrightText: © 2019-2021 Synthetix
```

Comment:
Let's update this one with the latest version.

Reply:
This mock is based on the latest version of https://github.com/makerdao/endgame-toolkit/blob/master/src/synthetix/StakingRewards.sol . I've just added back the notice that had been removed.

Reply:
Hmm ok, this doesn't include the constructor `require` change and some other minor comment changes but yeah the PR wasn't merged yet.

Reply:
Ah yes right, I'll include these changes as well